docs/providers/aws/guide/variables.md

```diff
@@ -41,12 +45,18 @@ functions:
       handler: handler.world
       events:
         - schedule: ${self:custom.globalSchedule}
+resources:
+  Outputs:
+    NewServiceExport:
+      Value: 'A Value To Export'
+      Export:
+        Name: ${self:custom.exportName}
 ```
 
 In the above example you're setting a global schedule for all functions by referencing the `globalSchedule` property in the same `serverless.yml` file. This way, you can easily change the schedule for all functions whenever you like.
 
 ## Referencing Environment Variables
-To reference environment variables, use the `${env:SOME_VAR}` syntax in your `serverless.yml` configuration file.
+To reference environment variables, use the `${env:SOME_VAR}` syntax in your `serverless.yml` configuration file.  `${env:}` references the environment variable hive.
```

Comment:
Of course.  Thanks for calling out the poor communication!  👍 
I'm thinking that I'll just be concrete and reference `process.env`.